var/spack/repos/builtin/packages/dd4hep/package.py

```diff
@@ -68,6 +70,10 @@ class Dd4hep(CMakePackage):
     depends_on('lcio', when="+lcio")
     depends_on('edm4hep', when="+edm4hep")
 
+    # See https://github.com/AIDASoft/DD4hep/pull/771
+    conflicts('^cmake@3.16:3.17.2', when='@1.15:1.16.1',
```

Comment:
```suggestion
    conflicts('^cmake@3.16:3.16.99', when='@1.15:1.16.1',
```
Because dd4hep should work with 3.17.X prior to PR 771 by already ignoring the python check for 3.17.1 and 3.17.2 (and 3.17.0 works anyway (I don't remember)?)

Reply:
Ah yes, good catch. Thanks. I seem to not have tested 3.17.0 in AIDASoft/podio#162 but the overall trend suggests that it should not work, so I have left that one in the conflict here.